noteable_magics/datasource_postprocessing.py

```diff
@@ -58,10 +58,34 @@ def postprocess_postgresql(
     # extra behavior into the driver as side-effect so that interrupting the
     # kernel does what we expect.
 
-    import psycopg2.extensions
-    import psycopg2.extras
+    _install_psycopg2_interrupt_fix()
 
```

Comment:
Get the fix for letting PG SQL cells be interruptable also present for CRDB, in case someone has a space where they gots only CRDB datasources and no PG ones (likely).

Unrelated to the topic of the ticket, but needed.